app/fight.py

```diff
@@ -0,0 +1,12 @@
+from app.KNIGHTS.knight import Knight
+
+
+def fight(knight1: "Knight", knight2: "Knight") -> None:
```

Comment:
Do not use quotes in the type annotation, remove them and use `Knight` class without them. Use `from __future__ import annotations` in case you face errors